src/V3EmitCImp.cpp

```diff
@@ -705,33 +768,43 @@ class EmitCTrace final : EmitCFunc {
             if (AstEnumDType* const enump = VN_CAST(nodep->skipRefToEnump(), EnumDType)) {
                 int enumNum = m_enumNumMap[enump];
                 if (!enumNum) {
+                    if (typesSplitNeeded()) {
+                        // Splitting file, so using parallel build.
+                        v3Global.useParallelBuild(true);
+                        // Close old file
+                        closeTypesFile();
+                        // Open a new file
+                        openNextTypesFile();
```

Comment:
```suggestion
                        closeTypesFile();
                        openNextTypesFile();
```

Seems pretty obvious as well named functions ;)